uv.lock

```diff
@@ -3299,7 +3344,7 @@ dependencies = [
 [[package]]
 name = "vllm-spyre"
 version = "0.0.0"
-source = { virtual = "." }
+source = { editable = "." }
```

Comment:
do you know how this was `virtual` before?

Reply:
🤔  Nope!
I still have a lot more to learn here